server/internal/core/application/covenantless.go

```diff
@@ -750,6 +750,11 @@ func (s *covenantlessService) UpdateTxRequestStatus(_ context.Context, id string
 }
 
 func (s *covenantlessService) SignVtxos(ctx context.Context, forfeitTxs []string) error {
+	start := time.Now()
+	defer func() {
+		end := time.Since(start)
+		fmt.Printf("SUBMIt FORFEIT TXS TOOK: %dµs", end.Microseconds())
```

Comment:
is fmt.Printf what we want? ie. is then controlled by the logrus log level? Also `SUBMIt` > `SUBMIT` if you want to keep upper case.

Reply:
it shoudl be a log level Trace, not even debug imho

Reply:
it's been removed